lib/model/migrations/20211027-01-add-purgeddAt-details-to-actees.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/opendatakit/central-backend/blob/master/NOTICE.
```

Comment:
You might want to go ahead and replace "opendatakit" and "getodk" now, since #427 has been approved.